core/src/main/resources/hudson/model/Messages.properties

```
@@ -324,6 +324,7 @@ UpdateCenter.PluginCategory.scm-related=Source Code Management related
 UpdateCenter.PluginCategory.security=Security
 UpdateCenter.PluginCategory.slaves=Agent Launchers and Controllers
 UpdateCenter.PluginCategory.test=Testing
+UpdateCenter.PluginCategory.theme=User Interface Themes
```

```suggestion
UpdateCenter.PluginCategory.theme=Themes
```

---

I've been thinking a bit about it. Wouldn't "Appearance" make sense? It would make it a broader category.

---

`UpdateCenter.PluginCategory.ui=User Interface` already exists.

---

Do you need a broader category though?

if you're after a theme don't you want a theme?

---

Didn't know about the UI category. Ignore my comment then.

---

```suggestion
UpdateCenter.PluginCategory.theme=UI Themes
```

But I like idea about appearance. I always wanted a category for the Chuck Norris Plugin

---

Which is `ui`? Or change it to `UpdateCenter.PluginCategory.ui=Appearance` if you like the term better?

---

I would prefer to have all of them, themes and others, within an Appearance category. But that's just a preference 🤷

---

I am not sire that UI === Appearance. 

* "Appearance" implies look&feel changes which do not impact the functionality
* UI is... something related to the User Interface. https://plugins.jenkins.io/ui/search/?labels=ui lists many plugins which offer functional changes

I do not see much sense to keep the "UI" category as is, but IMHO it is a separate discussion.

Right now I suggest to introduce a new "appearance" label, WDYT @daniel-beck @fqueiruga @timja ?

---

imo if I want to find a theme, I search for a theme.
I don't want to find the chuck norris plugin 😄

---

Login themes? Blue Ocean themes? Dunno.
Will change it to UI Themes